qt5-meta-toolchain does not build of no opengl

This is the error for a board where opengl is not in DISTRO_FEATURES

```
...
| CMake Error at Source/cmake/OptionsQt.cmake:529 (message):
|   Qt with OpenGL support is required for ENABLE_OPENGL
| Call Stack (most recent call first):
|   Source/cmake/WebKitCommon.cmake:50 (include)
|   CMakeLists.txt:137 (include)

```
The SDK builds okay for boards where opengl is in DISTRO_FEATURES.

Using this patch to get around it.

```
index d020cc3..2f8d8a2 100755
--- a/recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb
+++ b/recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb
@@ -9,7 +9,7 @@ PACKAGEGROUP_DISABLE_COMPLEMENTARY = "1"

 # Requires Ruby to work
 USE_RUBY = " \
-    qtwebkit-dev \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'qtwebkit-dev', '', d)} \
 "

 # Requires Wayland to work

```

--- a/recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb
+++ b/recipes-qt/packagegroups/packagegroup-qt5-toolchain-target.bb
@@ -9,7 +9,7 @@ PACKAGEGROUP_DISABLE_COMPLEMENTARY = "1"
 
 # Requires Ruby to work
 USE_RUBY = " \
-    qtwebkit-dev \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'qtwebkit-dev', '', d)} \
 "
 
 # Requires Wayland to work

